components/case-studies/CaseStudiesHeadingContent.tsx

```diff
@@ -10,15 +10,15 @@ const CaseStudiesHeading = () => (
       duration={1.75}
       className="mt-[4.4rem] w-full bg-white-800 px-[1.5rem] pt-[3rem] dark:bg-black-300"
     >
-      <h1 className="pb-[3rem] text-center text-[2.3rem] font-bold leading-[3.0rem] tracking-[-0.42px] text-black-200 dark:bg-black-300 dark:text-white-900 md:text-[4rem]   md:leading-[5.2rem] md:tracking-[-0.64px]">
+      <h1 className="pb-[3rem] text-center text-[2.625rem] font-bold leading-[3.02rem] tracking-[-0.42px] text-black-200 dark:bg-black-300 dark:text-white-900 md:text-[4rem] md:leading-[5.2rem] md:tracking-[-0.64px]">
```

Comment:
Please don't mix pixels and rem. You should use one or the other. Please try to look through your code and update it where applicable. Thanks!